### ISSUE TITLE
About creating my own data set

### ISSUE THREAD

**Issue by jiedia** [zhulf0804/NgeNet#13]
I want to add my own dataset based on the 3DMatch dataset. How do I create it? In 3DMatch data file, the training set and the test set are both a.txt file and a.pth file. What is the matrix in the.txt file? I also want to know where to store the conversion matrix between source point cloud and target point cloud (for point cloud registration training)?

**Comment by qsisi** [comment]
Different approaches have their own style of creating datasets based on the [original 3DMatch](https://vision.princeton.edu/projects/2016/3DMatch/), for example, FCGF, D3Feat, PREDATOR... etc. 
For convenience, one can directly use the data processed by PREDATOR, for training and test, or you can create a dataset on your own from [original 3DMatch](https://vision.princeton.edu/projects/2016/3DMatch/), which you could take a look at this:
[https://github.com/qsisi/3DMatch_Registration_Dataset](https://github.com/qsisi/3DMatch_Registration_Dataset)

**Comment by wmrenr** [comment]
> I want to add my own dataset based on the 3DMatch dataset. How do I create it? In 3DMatch data file, the training set and the test set are both a.txt file and a.pth file. What is the matrix in the.txt file? I also want to know where to store the conversion matrix between source point cloud and target point cloud (for point cloud registration training)?

Could you please  successfuly create your  own dataset based on the 3DMatch dataset?
